node/repo/importmgr/mgr.go

```diff
@@ -35,11 +35,11 @@ const (
 	LFileStoreCARv2FilePath = "CARv2Path" // path of the full CARv2 file or a CARv2 file that can serve as the backing store for a Filestore.
 )
 
-func New(ds datastore.Batching, repoPath string) *Mgr {
+func New(ds datastore.Batching, tmpDir string) *Mgr {
 	return &Mgr{
-		repoPath: repoPath,
-		ds:       datastore.NewLogDatastore(namespace.Wrap(ds, datastore.NewKey("/stores")), "storess"),
-		counter:  shared.NewTimeCounter(),
+		tmpDir:  tmpDir,
```

Comment:
Technically speaking, this doesn't need to be a tempdir. Can we call `baseDir`?

Reply:
It's only used in NewTempFile(), but in any case happy to change the name